src/pattern-matching/Structs.md

```diff
@@ -0,0 +1,21 @@
+---
+minutes: 8
```

Comment:
The minutes here should be split between the two pages: perhaps 4 minutes for each?

Reply:
Ok.